port/unix/omrsysinfo.c

```diff
@@ -453,24 +455,45 @@ typedef struct OMRCgroupMetricInfoElement {
 	BOOLEAN isValueToBeChecked;
 } OMRCgroupMetricInfoElement;
 
-static struct OMRCgroupMetricInfoElement oomControlMetricElementList[] = {
+static struct OMRCgroupMetricInfoElement oomControlMetricElementListV1[] = {
 	{ "OOM Killer Disabled", "oom_kill_disable", NULL, FALSE },
 	{ "Under OOM", "under_oom", NULL, FALSE }
 };
 
-static struct OMRCgroupMetricInfoElement cpuStatMetricElementList[] = {
+static struct OMRCgroupMetricInfoElement memEventsMetricElementListV2[] = {
+	{ "Approached memory limit count", "max", NULL, FALSE },
+	{ "Reached memory limit count", "oom", NULL, FALSE }
+};
+
+static struct OMRCgroupMetricInfoElement swapEventsMetricElementListV2[] = {
+	{ "Approached swap limit count", "max", NULL, FALSE },
+	{ "Swap alloc failed count", "fail", NULL, FALSE }
+};
+
+static struct OMRCgroupMetricInfoElement cpuStatMetricElementListV1[] = {
 	{ "Period intervals elapsed count", "nr_periods", NULL, FALSE },
 	{ "Throttled count", "nr_throttled", NULL, FALSE },
 	{ "Total throttle time", "throttled_time", "nanoseconds", FALSE }
 };
 
+static struct OMRCgroupMetricInfoElement cpuMaxMetricElementListV2[] = {
+	{ "CPU Quota", (char *)CGROUP_METRIC_NO_KEY, "microseconds", TRUE },
+	{ "CPU Period", (char *)CGROUP_METRIC_NO_KEY, "microseconds", FALSE }
```

Comment:
We should add a comment about using `CGROUP_METRIC_NO_KEY`. If the above order is changed, then a wrong value will be retrieved for each metric. Also, we should provide instructions (with an example) if `CGROUP_METRIC_NO_KEY` can be mixed/used with a `string metric key`.

Reply:
> Also, we should provide instructions (with an example) if CGROUP_METRIC_NO_KEY can be mixed/used with a string metric key.

Not sure what you mean. CGROUP_METRIC_NO_KEY cannot be used for files containing keyed values and no-keyed values (which don't exist in the cgroup file system)

Reply:
I would like the usage of `CGROUP_METRIC_NO_KEY` to be documented. Dos and don'ts for using `CGROUP_METRIC_NO_KEY` in order to remove the ambiguity around its usage. 

The below scenario may not exist in any of the `cgroup` files. Is it possible to parse it with the current mechanism?
```
key1 value1
no_key_value1 no_key_value2
key2 value2
```

Reply:
No it is not possible